File: qe-selenium-python/activityThree.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from time import sleep

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/login-form")
    print(driver.title)

    sleep(2)

    driver.find_element(By.XPATH, "//input[@id='username']").send_keys("admin")
    sleep(2)
    driver.find_element(By.XPATH, "//input[@id='password']").send_keys("password")
    sleep(2)
    driver.find_element(By.XPATH, "//button[text()='Submit']").click()
    sleep(2)
    driver.quit()

File: qe-selenium-java/src/main/java/demos/ActivityThree.java
package demos;

import org.openqa.selenium.firefox.*;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class ActivityThree {
    public static void main(String[] args) throws InterruptedException {
        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/login-form");
        System.out.println(driver.getTitle());

        Thread.sleep(2000);

        driver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");
        driver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");

        Thread.sleep(2000);
        driver.findElement(By.xpath("//button[text()='Submit']")).click();
        driver.quit();

    }

}